tee/prototype/enclave.go

```diff
@@ -59,6 +58,17 @@ func NewEnclave(wallet accounts.Wallet) *Enclave {
 	}
 }
 
+func NewEnclaveWithAccount(wallet accounts.Wallet, account accounts.Account) *Enclave {
+	return &Enclave{
```

Comment:
The latest version or `NewEnclave` looks different, better don't copy the code from `NewEnclave` but call `NewEnclave` and then set the account field manually. Otherwise we have to keep copied code in sync.

Reply:
ok, done